### PR TITLE
Move communal backend instruction in DEVELOPER.md

### PR DESCRIPTION
**Pull request by roypaulin**
This PR moves communal backend instruction, that was previously in a confluence page, in DEVELOPER.md.

### CODE REVIEW

**Comment by roypaulin** [comment]
I did not confluence page links. I was not sure if it is something that should be exposed in the repo.

File: DEVELOPER.md [hunk 1]
```diff
@@ -260,9 +260,110 @@ Here are the steps on how to override them:
...
+               BUCKET_OR_CLUSTER  | If not using HA namenode, you can put in the name node host.  If you are using an HA namenode, include the nameservice name. | hdp31ns or  hdfs-namenode-0.hdfs-namenode.kuttl-e2e-hdfs.svc.cluster.local:50070                              
+               PATH_PREFIX | Fill this in if you want to store the database in some subdirectory within the HDFS cluster. | / or /user/mspilchen/                          
+               HADOOP_CONF_CM  | Name of the config map that contains the hadoop config director (i.e. /etc/hadoop).  This is the name of the configMap that you created at step 2.  This can be left blank if not using HA namenode. | hadoop-conf
+               HADOOP_CONF_NAMESPACE  | The k8s namespace that HADOOP_CONF_CM is in.  This can be blank if HADOOP_CONF_CM is blank. | eng-p9
+               COMMUNAL_EP_CERT_SECRET  | If using swebhdfs:// scheme, this is the name of the secret created at step 1 above. | hadoop-certs
+               COMMUNAL_EP_CERT_NAMESPACE  | If using swebhdfs://, the namespace that the Secret from step1 was created in. | certs
+
+      5. PURESTORAGE
```
Comment (spilchen) [inline DEVELOPER.md:348]: I suggest we remove the pureStorage section entirely.  Showing how to connect to our internal system probably isn't very beneficial.

File: DEVELOPER.md [hunk 2]
```diff
@@ -260,9 +260,110 @@ Here are the steps on how to override them:
 
 4. Setup the commmunal endpoint.
 
+      1. AWS S3 BUCKET
+
+         If you have an MicroFocus account for AWS, you can configure your environment so that it uses AWS instead of minio.  There are pre-existing buckets setup for developers to use.  Here are the steps to set that up:
```
Comment (spilchen) [inline DEVELOPER.md:265]: Rather than say MicroFocus account, lets generalize it.  This is how we connect to any AWS endpoint.

**Review by spilchen — COMMENTED** [review]
Looks good.  Just had some comments.

File: DEVELOPER.md (hunk 1 above)
Comment (roypaulin) [reply DEVELOPER.md:348]: yeah I thought so. I will put that on the communal storage confluence page

File: DEVELOPER.md [hunk 3]
```diff
@@ -262,7 +262,7 @@ Here are the steps on how to override them:
 
       1. AWS S3 BUCKET
 
-         If you have an MicroFocus account for AWS, you can configure your environment so that it uses AWS instead of minio.  There are pre-existing buckets setup for developers to use.  Here are the steps to set that up:
+         If you have an AWS account, you can configure your environment so that it uses AWS instead of minio.  There are pre-existing buckets setup for developers to use.  Here are the steps to set that up:
```
Comment (spilchen) [inline DEVELOPER.md:265]: > There are pre-existing buckets setup for developers to use.

Maybe it should say, "The s3 bucket must already be created prior to running the tests".

Comment (roypaulin) [reply DEVELOPER.md:265]: Ah this for any developer so it has to be the most general possible. right?

Comment (roypaulin) [reply DEVELOPER.md:265]: So "pre-existing buckets setup for developers" are only for internal use?

Comment (spilchen) [reply DEVELOPER.md:265]: Right.  That sentence relates to the internal buckets Vertica uses for testing.  Best to have it more general.

**Review by spilchen — APPROVED** [review]
Looks good.  Thanks for revising.

**Pull request merged**